### ISSUE TITLE
When label value is empty/undefined hide label

### ISSUE THREAD

**Issue by bryanspears** [AmpersandJS/ampersand-select-view#53]
I'm not sure how consistent this module wants to be with ampersand-input-view, but I like that input-view hides the label if the content is empty.

https://github.com/AmpersandJS/ampersand-input-view/blob/master/ampersand-input-view.js#L36

This is mainly a question of consistency rather than a hard bug report. :smile:

**Comment by cdaringe** [comment]
right right. hmm.  Sheesh I'm not sure.  @lynnandtonic, what do you think?

**Comment by lynnandtonic** [comment]
Not sure what other implications there might be, but I'm a +1 for consistency.

**Comment by e2jk** [comment]
I'm just hitting this as well. I'm integrating a form at the bottom of a table to add elements to that table, the table's column already contains the "labels" so I don't need the `<select>` to show its own label. The inputs are showing fine, but without an explicit label for the select the name of that element is displayed nonetheless:

![capture du 2016-01-23 00 40 43](https://cloud.githubusercontent.com/assets/229379/12526090/fad46400-c169-11e5-99eb-eafdc5556685.png)

**Comment by e2jk** [comment]
That was actually pretty easy to fix, just replace
```
        if (opts.label === undefined) {
            this.label = this.name;
        } else {
            this.label = opts.label;
        }
```
by
```
        if (opts.label !== undefined) {
            this.label = opts.label;
        }
```

and now this is displayed:
![capture du 2016-01-23 00 46 50](https://cloud.githubusercontent.com/assets/229379/12526158/d076b70c-c16a-11e5-8ad8-c7ffd43654f9.png)

**Comment by e2jk** [comment]
Addressed in PR #62.
